jpsonic-main/src/main/java/com/tesshu/jpsonic/domain/JpMediaFileComparator.java

```diff
@@ -98,21 +104,14 @@ private int compareAlbumAndNotAlbum(MediaFile a, MediaFile b) {
         return 0;
     }
 
-    @SuppressWarnings("PMD.ConfusingTernary") // false positive
-    private int compareDirectory(MediaFile a, MediaFile b) {
-        int n;
+    int compareDirectory(MediaFile a, MediaFile b) {
```

Comment:
Comparison between directories. Removed unnecessary comparisons that caused conflicts.